qiime/util.py

```diff
@@ -530,7 +530,7 @@ def get_generated_by_for_biom_tables():
 
 
 def write_biom_table(biom_table, biom_table_fp, compress=True,
-                     write_hdf5=HAVE_H5PY):
+                     write_hdf5=HAVE_H5PY, type='OTU table'):
```

Comment:
could this be called ``table_type`` to match the cli and because ``type`` is a builtin in python?